hc/accounts/tests/test_check_token.py

```diff
@@ -1,5 +1,7 @@
 from django.contrib.auth.hashers import make_password
 from hc.test import BaseTestCase
+from .. import views
```

Comment:
Fix import order to adhere to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) standard